### DISCUSSION TITLE
Magic3D refine on Colab

### DISCUSSION THREAD

**Discussion by vilo9595** [threestudio-project/threestudio#408] [Q&A]
The refinement stage of Magic3D doesn't seem to work when trying it in colab. I tried tried to get it to work and also tried a new environment because I played around with other custom models before. Does it work for everyone else?

**Comment by pavankay** [comment]
I was doing magic 123 and got this error [F glutil.cpp:338] eglInitialize() failed Aborted (core dumped) was this the same one you got

**Comment by vilo9595** [comment]
`!python launch.py --config configs/magic3d-refine-sd.yaml --train --gpu 0 system.prompt_processor.prompt="$prompt" system.geometry_convert_from=/content/threestudio/outputs/magic3d-coarse-if/An_orange_poison-dart_frog_sitting_on_a_water_lily@20240213-141944/ckpts/last.ckpt system.renderer.context_type=cuda system.exporter.context_type=cuda
`

This works
